go/ct/rlz/domains.go

```diff
@@ -374,7 +374,7 @@ func (d gasDomain) Samples(a vm.Gas) []vm.Gas {
 }
 
 func (gasDomain) SamplesForAll(as []vm.Gas) []vm.Gas {
-	res := []vm.Gas{0, st.MaxGas}
+	res := []vm.Gas{0, 100, 200, st.MaxGas}
```

Comment:
Since almost all rules have a condition on the gas this change will almost double the amount of test cases we produce. It should be investigated whether a change with less performance impact would be sufficient.

Reply:
this is a great observation, I thought it would have an impact but not as big as it did. After a little experimenting, I removed the `100`, but in order to hit those edge cases an intermediate value independent from the rules' base cost is needed.